virttest/utils_test/__init__.py

```diff
@@ -921,9 +921,10 @@ def get_results(test_path, result_path, session):
         # result info tarball to host result dir
         results_tarball = os.path.join(test_path, "results.tgz")
         compress_cmd = "cd %s && " % result_path
-        compress_cmd += "tar cjvf %s ./*" % results_tarball
+        compress_cmd += "tar cjvf%s" % results_tarball
```

Comment:
Please leave a blank space between options 'cjvf' and results_tarball

Reply:
Other changes looks Ok to me.

Reply:
Fixed.